src/assets/styles/10-settings/_breakpoints.scss

```diff
@@ -6,9 +6,23 @@
 // 2019-11-11
 // ----------------------------------
 
+// ----------------------------------
+// Breakpoints
+// sm: 768px
+// md: 1024px
+// lg: 1200px
+// xl: 1344px
+// ----------------------------------
+
 $breakpoints: (
-  sm: 500px,
-  md: 738px,
-  lg: 920px,
-  xl: 1100px
+  sm: 48rem,
+  md: 64rem,
+  lg: 75rem,
+  xl: 84rem
 ) !default;
+
+@mixin breakpoint($breakpoint) {
```

Comment:
Thoughts on using sass-mq going forwards? https://github.com/sass-mq/sass-mq

Gives us a more robust media-queries and we don't have to maintain it ourselves 😄

Happy to raise an issue/PR for this separately though.

Reply:
Absolutely let's use it. Just include it with next style update you do, no need for another ticket